tests/lib/prepare-uc.sh

```diff
@@ -7,10 +7,10 @@ set -x
 . "$TESTSLIB/prepare-utils.sh"
 
 # install dependencies
-install_core22_deps
+install_core24_deps
 
 # download snaps required for us to build the image
-download_core22_snaps "$SNAP_BRANCH"
+download_core24_snaps "$SNAP_BRANCH"
```

Comment:
download_base_snaps or similar

Reply:
Well, given the implementation is only for core24, I am not sure I want to change that.

Reply:
Ok, that's fine